### ISSUE TITLE
using superset 1.1.0 suddenly not able to start superset

### ISSUE THREAD

**Issue by tejaskumar-cm** [apache/superset#15524]
I am using below version
Python 3.8.10
apache-superset==1.1.0

It was working fine. But suddenly got below error:

logging was configured successfully
INFO:superset.utils.logging_configurator:logging was configured successfully
/home/ubuntu/apache-superset/venv/lib/python3.8/site-packages/flask_caching/__init__.py:201: UserWarning: Flask-Caching: CACHE_TYPE is set to null, caching is effectively disabled.
  warnings.warn(

Failed to create app
Traceback (most recent call last):
  File "/usr/lib/python3.8/pkgutil.py", line 415, in get_importer
    importer = sys.path_importer_cache[path_item]
KeyError: PosixPath('/home/ubuntu/apache-superset/venv/lib/python3.8/site-packages/superset/db_engine_specs')

I tried to start the superset server again. But getting same above error. We haven't changed any config or package file. Running superset server in ec2 suddenly broke.

![image](https://user-images.githubusercontent.com/57861380/124274855-65217680-db5f-11eb-9fff-90f57ea4cab6.png)

**Comment by ag-supersetuser** [comment]
Hi @tejaskumar-cm 
We have had exactly the same issue since midnight BST/GMT 01 July.
We have spent all yesterday and today trying to fix this. 
On a test environment we have created a new install which threw up a brand new release, v1.2.0
see here: https://github.com/apache/superset/releases
After running a PIP upgrade, and then installing Superset through the PIP commands, our test version is now live and up and running.
Seems like something was broken somewhere, no information was given about this and it has been fixed and released in this v1.2.0
Hope this helps you as we have been pulling our hair out for two days on this!
